### PR TITLE
Added bpytop for macOS and iOS.

### PR DESCRIPTION
**Pull request by DaOfficialRolex**

### CODE REVIEW

**Comment by DaOfficialRolex** [comment]
It should be noted that this doesn't work without python-psutil. This will be converted into a draft for the time being until that psutil is added and merged into as2 branch.

**Comment by Cryptiiiic** [comment]
This pull works out of the box on iOS given that you have python3-psutil.
image

File: bpytop.mk [hunk 1]
```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += bpytop
+BPYTOP_VERSION := 1.0.63
+DEB_BPYTOP_V   ?= $(BPYTOP_VERSION)
+
+bpytop-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE)  https://github.com/aristocratos/bpytop/archive/refs/tags/v$(BPYTOP_VERSION).tar.gz
```
Comment (Diatrus) [inline bpytop.mk:10]: GITHUB_ARCHIVE? :)

File: bpytop.mk [hunk 2]
```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += bpytop
+BPYTOP_VERSION := 1.0.63
```
Comment (Diatrus) [inline bpytop.mk:6]: There's a newer version now

**Review by Diatrus — CHANGES_REQUESTED** [review]

**Pull request merged**